src/simplewallet/simplewallet.cpp

```diff
@@ -244,6 +258,7 @@ namespace
   const char* USAGE_THAW("thaw <key_image>");
   const char* USAGE_FROZEN("frozen <key_image>");
   const char* USAGE_NET_STATS("net_stats");
+  const char* USAGE_PUBLIC_NODES("public_nodes");
```

Comment:
Maybe do the same for the other three new commands (`rpc_payment_info`, `start_mining_for_rpc`, `stop_mining_for_rpc`)?